src/main/java/fr/insee/genesis/controller/rest/responses/RawResponseController.java

```diff
@@ -69,6 +77,73 @@ public ResponseEntity<String> saveRawResponsesFromJsonBody(
         return ResponseEntity.status(201).body(String.format(SUCCESS_MESSAGE,interrogationId));
     }
 
+    @Operation(summary = "Save lunatic json data from one interrogation in Genesis Database (with json " +
+            "schema validation)")
+    @PutMapping(path = "/lunatic-json")
```

Comment:
Maybe a name more explicit to differentiate it from the old one (add v2 or something else)